modules/common/tls/tls.go

```diff
@@ -67,6 +68,13 @@ const (
 
 	// TLSHashName - Name of the hash of hashes of all cert resources used to identify a change
 	TLSHashName = "certs"
+
+	// AdditionalSubjectNames - Comma separated list of additionalSubjectNames
+	// that should be passed to the CertificateRequest
+	AdditionalSubjectNames = "additionalSubjectNames"
```

Comment:
to align with the above, could you name it `AdditionalSubjectNamesKey`

Reply:
ack